src/Log/LogService.php

```diff
@@ -211,30 +211,34 @@ public function setPath($logPath)
     public function getDefaultLog()
     {
         if (array_key_exists(self::DEFAULT_LOG.'.log', $this->loggers)) {
```

Comment:
The behavior of adding '.log' to the name of the logger needs to be explained. Please add more function comments. I am not saying the behavior is necessarily wrong, but it triggers warning signals. Why do we add '.log' internally. It is something we have to do in various places, like here and at least where the logger is added to $this->loggers. This  makes bugs easy if somebody does not pay attention. So, are there good reasons and if so they need to be documented.

Reply:
We decided that keeping the loggers array as an associated array with filename as keys is a good idea in one of the previous PRs. That's the reason why we have to add '.log' while looking for a key. I can document the same in the comments.

Reply:
In that case we need to pay attention to the following. In how many places would I have to make changes if I wanted to just use the name in `loggers`. Also we can configure the `file` for a logger. That means the name of the logger and the file name do not have to match. The most obvious example is the `default` logger, that most of the time writes to `opus.log`, but sometimes logs into `opus-console.log`. So the thinking in the previous PR was that we might need to have a mapping between loggers and log files in order to prevent creating loggers for the same file twice. However we definitely need to map between names and loggers. So we probably need two arrays.

Reply:
Yes you are right. We need to map the loggers with an array of logfile names. Will work on it.